llvm/test/CodeGen/RISCV/rvv/fixed-vectors-vw-web-simplification.ll

```diff
@@ -58,3 +58,26 @@ define <2 x i16> @vwmul_v2i16_multiple_users(ptr %x, ptr %y, ptr %z) {
   %i = or <2 x i16> %h, %g
   ret <2 x i16> %i
 }
+
+; FIXME: We should have a vsext.vl instead of vzext.vl.
```

Comment:
I think this FIXME is completed?

Reply:
Weird. I remember changing it. I’ll fix it. Thanks!